addons/hr_recruitment/static/src/js/tours/hr_recruitment.js

```diff
@@ -6,36 +6,97 @@ var tour = require('web_tour.tour');
 }, {
     trigger: '.o_app[data-menu-xmlid="hr_recruitment.menu_hr_recruitment_root"]',
-    content: _t('Want to <b>start recruiting</b> like a pro? <i>Start here.</i>'),
+    content: _t("Let's have a look at how to <b>improve</b> your <b>hiring process</b>."),
     position: 'bottom',
     edition: 'enterprise'
 }, {
     trigger: ".o-kanban-button-new",
-    extra_trigger: '.o_hr_recruitment_kanban',
-    content: _t("Click here to create a new job position."),
-    position: "bottom"
+    content: _t("Create your first Job Position."),
+    position: "bottom",
+    width: 195
+}, {
+    trigger: ".o_job_name",
+    extra_trigger: '.o_hr_job_simple_form',
+    content: _t("What do you want to recruit today? Choose a job title..."),
+    position: "right"
+}, {
+    trigger: ".o_job_alias",
+    extra_trigger: '.o_hr_job_simple_form',
+    content: _t("Choose an application email."),
+    position: "right"
+}, {
+    trigger: '.o_create_job',
+    content: _t('Let\'s create the position. An email will be setup for applications, and a public job description, if you use the Website app.'),
+    position: 'bottom',
+    run: function (actions) {
+        actions.auto('.modal:visible .btn.btn-primary');
+    },
 }, {
     trigger: ".oe_kanban_action_button",
     extra_trigger: '.o_hr_recruitment_kanban',
-    content: _t("Let\'s have a look at the <b>applications pipeline</b> for this job position."),
+    content: _t("Let\'s have a look at the applications pipeline."),
+    position: "bottom"
+}, {
+    trigger: ".o_copy_paste_email",
+    content: _t("Copy this email address, to paste it in your email composer, to apply."),
     position: "bottom"
 }, {
     trigger: ".breadcrumb-item:not(.active):last",
     extra_trigger: '.o_kanban_applicant',
-    content: _t("Use the breadcrumbs to <b>go back to the dashboard</b>."),
+    content: _t("Let’s go back to the dashboard."),
     position: "bottom"
 }, {
-    trigger: ".o_job_alias",
+    trigger: ".oe_kanban_action_button",
     extra_trigger: '.o_hr_recruitment_kanban',
-    content: _t("Try to send an email to this address, it will create an application automatically."),
+    content: _t("<b>Did you apply by sending an email?</b> Check incoming applications."),
     position: "bottom"
+}, {
+    trigger: ".oe_kanban_card",
+    extra_trigger: '.o_kanban_applicant',
+    content: _t("<b>Drag this card</b>, to qualify him for a first interview."),
+    position: "bottom",
+    run: "drag_and_drop .o_kanban_group:eq(1) ",
+}, {
+    trigger: ".oe_kanban_card",
+    extra_trigger: '.o_kanban_applicant',
+    content: _t("<b>Click to view</b> the application."),
+    position: "bottom"
+}, {
+    trigger: ".o_Chatter .o_ChatterTopbar_buttonSendMessage",
+    extra_trigger: '.o_applicant_form',
+    content: _t("<div><b>Try to send an email</b> to the applicant.</div><div><i>Tips: All emails sent or received are saved in the history here</i>"),
+    position: "bottom"
+}, {
```

Comment:
Shouldn't we have an extra step here asking the user to write on the textarea?

Reply:
No, we've already discussed with FLL about this

Reply:
Ok, but then what is the reasoning for not doing it? Because it seems pretty counter-intuitive to me to onboard someone by telling him to click on a button that doesn't work.

Reply:
Hello,
2 Reasons :
First one, we try to have as steps as possible
Secondly, when we click on the button "Send messages" our cursor is already in the message box where we have "Write something". So, if we add a step, it seems like a duplicate